content/references/proptypes.md

```diff
@@ -78,6 +80,45 @@ export const defaultConfig = {
 
 Merchant edits to color fields are stored in RGBA format.
 
+### date
+
+![Date Picker](proptype-date.png)
+
+```javascript
+date: {
+  label: 'Date',
+  type: ElementPropTypes.date
+}
+
+export const defaultConfig = {
+  date: new Date('March 16, 2020')
+}
+```
+
+### dateRange
+
+![DateRange Picker](proptype-dateRange.png)
+
+```javascript
+dateRange: {
+  label: 'Date Range',
+  type: ElementPropTypes.dateRange
+}
+
+const today = new Date();
+const nextWeek = new Date();
+nextWeek.setDate(nextWeek + 7);
+
+export const defaultConfig = {
+  dateRange: [
+    new today,    // start date
+    new nextWeek  // end date
+  ]
+}
```

Comment:
Did we confirm this works as we hoped (that it always starts the date range at the date of adding the block to the page)?

Reply:
Correct, the date is set when the block is added to the page.